src/app/workbench/WorkbenchView.tsx

```diff
@@ -25,12 +25,10 @@ export function WorkbenchView({
   removeCallback: (path: MosaicPath) => void;
 }) {
   const ordino = useAppSelector((state) => state.ordino);
-  // eslint-disable-next-line react-hooks/exhaustive-deps
-  const views = useMemo(() => () => getVisynView(ordino.workbenches[workbenchIndex].entityId), []);
-  const { value } = useAsync(views, []);
+  const { value } = useAsync(getVisynView, [ordino.workbenches[workbenchIndex].entityId]);
```

Comment:
bad variable name: value.

Reply:
ya good point, i actually do this a lot when using that useAsync where I dont bother renaming the returned value. I'll get out of that habit, renamed it to visynViews here.